web/src/lib/domain-logic/isGenericIndustry.ts

```diff
@@ -0,0 +1,3 @@
+export const isGenericIndustry = (sectorId: string | undefined): boolean => {
```

Comment:
[sand] If it's `string | undefined`, shouldn't this parameter be optional?

Reply:
done